docs/figures/auth.rest

```diff
@@ -0,0 +1,44 @@
+======================================
+Scistream Client Authentication Workflow
+======================================
+
+Making a request without Local Credentials
+------------------------------------------
+
+**Case A: Scistream Server Does Not Require Authentication (Default Case)**
+
+- *Server Command*::
```

Comment:
Should we change this to *Command for initializing server:*?

Reply:
I also modified this, so this comment is now outdated